knowledge-base/display-week-number-scheduler-header-maui.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Displaying Week Number in Scheduler HeaderTextFormat in .NET MAUI
```

Comment:
I suggest reducing the length as .NET MAUI is added by default during the build.
```suggestion
title: Displaying Week Numbers in the Scheduler's HeaderTextFormat
```